metaverse_modules/sonic-boom/index.js

```diff
@@ -7,7 +7,7 @@ const baseUrl = import.meta.url.replace(/(\/)[^\/\\]*$/, '$1');
 
 export default () => {
   const app = useApp();
-  const localPlayer = useLocalPlayer();
+  const localPlayer = app.player ?? useLocalPlayer();
```

Comment:
If the player is being passed in, it should be a component (`playerId`). If it's `null` it can mean local, otherwise we can look it up as a remote player.

In general the interface to programming apps should be components only.